File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/subscope/RepeatUsageDependency_ViewModelTestEnvironmentImpl.cpp
#include "RepeatUsageDependency_ViewModelTestEnvironmentImpl.h"
#include <memory>
#include "RepeatUsageDependency_ViewModel.h"
#include "MessageBus.h"
#include "dependencies/LoggerSpy.h"

namespace dependencies::subscope
{
  void RepeatUsageDependency_ViewModelTestEnvironmentImpl::Init()
  {
  }
  std::shared_ptr<RepeatUsageDependency_ViewModel> RepeatUsageDependency_ViewModelTestEnvironmentImpl::BuildSut()
  {
    return nullptr;
  }
  std::shared_ptr<MessageBus> RepeatUsageDependency_ViewModelTestEnvironmentImpl::getMessageBus()
  {
    return nullptr;
  }
  std::shared_ptr<LoggerSpy> RepeatUsageDependency_ViewModelTestEnvironmentImpl::getLogger()
  {
    return nullptr;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/subscope/RepeatUsageDependency_ViewModelTestEnvironmentImpl.cs
/// <filename>
///     RepeatUsageDependency_ViewModelTestEnvironmentImpl.cs
/// </filename>
using dependencies.subscope;
using dependencies;

namespace dependencies.subscope
{
    public class RepeatUsageDependency_ViewModelTestEnvironmentImpl : dependencies.subscope.RepeatUsageDependency_ViewModelTestEnvironment
    {
        public override void Init()
        {
        }

        public override dependencies.subscope.RepeatUsageDependency_ViewModel BuildSut()
        {
            return null;
        }

        public override dependencies.subscope.MessageBus getMessageBus()
        {
            return null;
        }

        public override dependencies.LoggerSpy getLogger()
        {
            return null;
        }
    }
}
